ui/components/app/transaction-alerts/transaction-alerts.js

```diff
@@ -27,11 +48,41 @@ const TransactionAlerts = ({
   const pendingTransactions = useSelector(submittedPendingTransactionsSelector);
   const t = useI18nContext();
 
+  ///: BEGIN:ONLY_INCLUDE_IN(blockaid)
+  const trackEvent = useContext(MetaMetricsContext);
+  const txParams = txData?.txParams || {};
+  const methodData = useSelector((state) =>
+    getKnownMethodData(state, txParams?.data),
+  );
+  ///: END:ONLY_INCLUDE_IN
+
+  ///: BEGIN:ONLY_INCLUDE_IN(blockaid)
+  const onClickSupportLink = useCallback(() => {
+    trackEvent({
+      category: MetaMetricsEventCategory.Transactions,
+      event: MetaMetricsEventName.ExternalLinkClicked,
+      properties: {
+        action: 'Confirm Screen',
+        legacy_event: true,
+        recipientKnown: null,
```

Comment:
can we omit this one as well? not sure what function this has or where the events are specified